lib/game.rb

```diff
@@ -1,12 +1,7 @@
 class Game
-  attr_reader :start_game
-
-  def initialize
-    @start_game =  "Welcome to BATTLESHIP"
-  end
-
   def intro
-   puts @start_game
```

Comment:
Great idea on removing the `initialize` and the `@start_game` instance method and consolidating it to the `intro` method.